charmhelpers/core/host.py

```diff
@@ -1075,3 +1075,32 @@ def install_ca_cert(ca_cert, name=None):
     log("Installing new CA cert at: {}".format(cert_file), level=INFO)
     write_file(cert_file, ca_cert)
     subprocess.check_call(['update-ca-certificates', '--fresh'])
+
+
+def get_system_env(key, default=None):
```

Comment:
To maintain mental consistency with `dict.get()`, I would return `None` rather than raise a `KeyError` error on a method that looks like `get_`.

Reply:
Good point @timClicks, the intention was to be able to require success, but I guess that can be done by the caller by checking the return rather than, perhaps surprisingly, raising a KeyError.